README.md

```diff
@@ -69,86 +61,86 @@ Please be willing to contribute if you can, instead of being ready to demand. Th
+If you have ideas, feedback, suggestions or comments remember to speak up and use your voice. You
+can add reports or request features and they will be considered.
 
 ## Why was the project started?
 
 Because raccoons are so adorable, aren't they? 🦝🦝🦝
 
-Joking apart, one of the main goals was to experiment with KMP and learn how to properly deal
-with the challenges of a multiplatform environment, and a medium-sized project like this was an
-ideal testing ground for that technology.
+Joking apart, one of the main goals was to experiment with KMP and learn how to properly deal with
+the challenges of a multiplatform environment, and a medium-sized project like this was an ideal
+testing ground for that technology.
 
-Secondly, as a means to dig deeper inside Lemmy's internals and become more humble and patient
-towards other apps whenever I found anything lacking or not implemented in an optimal way (according
-to my taste).
+Secondly, I felt that the Android ecosystem of Lemmy apps was a little "poor" with few native apps
+(fewer open source), while the "market" is dominated by iOS and cross-platform clients.
+I️ ❤️ Kotlin, I ❤️ Free and Open Source Software and I ❤️ native app development, so there was a niche
+that could be filled.
 
-In the third place, I felt that the Android ecosystem of Lemmy apps was a little "poor" with few
-native apps (fewer open source), while the "market" is dominated by iOS and cross platform clients.
-I️ ❤️ Kotlin, I ❤️ Free and Open Source Software and I ❤️ native app development, so there was a
-"niche" that needed to be filled.
+Developing a new client was an opportunity to add all the good features that were "scattered" across
+different apps, e.g. the feature richness of [Liftoff](https://github.com/liftoff-app/liftoff),
+themulti-community feature of [Summit](https://github.com/idunnololz/summit-for-lemmy) and the
+polished UI of the really great [Thunder](https://github.com/thunder-app/thunder) and so on.
+This app tries to be configurable enough to make users feel "at home" and choose what they want,
+while at the same time having a not too cluttered interface.
 
-Moreover, developing a new client was an opportunity to add all the features that I needed and that
-were "scattered" across different apps, e.g. I liked the feature richness
-of [Liftoff](https://github.com/liftoff-app/liftoff) (e.g. the possibility to explore all the
-communities of an external instance in guest mode), the multi-community feature of
-[Summit](https://github.com/idunnololz/summit-for-lemmy) and the polished UI of the really great
-[Thunder](https://github.com/thunder-app/thunder) and I wished I could have them all in the same
-app.
+In the third place, this app has been a means to dig deeper inside Lemmy's internals and become more
+humble and patient towards other apps because there are technical difficulties in having to deal
+with a platform likeLemmy.
```

Comment:
One missing space, but otherwise it all looks beautiful! 
I like the other changes.

Reply:
Ok, done!